packages/guides/src/Compiler/NodeTransformers/MenuNodeAddSubDocumentsTransformer.php

```diff
@@ -69,6 +74,18 @@ public function supports(Node $node): bool
         return $node instanceof TocNode || $node instanceof NavMenuNode;
     }
 
+    private function isInRootline(DocumentEntryNode $menuEntry, DocumentEntryNode $currentDoc): bool
+    {
+        return $menuEntry->getFile() === $currentDoc->getFile()
```

Comment:
I would like to split this line into multiple return statements. This will make the code more readable. 

if (...)  {
   return true;
}

if ($current->getParent() === null) {
   return false;
}

return false;